gateway/src/main/resources/config/application-dev.yml

```diff
@@ -0,0 +1,119 @@
+# ===================================================================
```

Comment:
(simmilar)
diff link: https://www.diffchecker.com/RfyVjNKn

Reply:
I think here the main problem is with the space counts, good catch